caliban_toolbox/reshape_data.py

```diff
@@ -216,11 +220,12 @@ def reconstruct_image_stack(crop_dir, verbose=True):
 
     # labels for each index within a dimension
     _, stack_len, _, _, row_len, col_len, _ = log_data['original_shape']
+    label_name = log_data['label_name']
     coordinate_labels = [log_data['fov_names'], range(stack_len), range(1),
-                         range(1), range(row_len), range(col_len), ['segmentation_label']]
+                         range(1), range(row_len), range(col_len), [label_name]]
 
     # labels for each dimension
-    dimension_labels = ['fovs', 'stacks', 'crops', 'slices', 'rows', 'cols', 'channels']
+    dimension_labels = ['fovs', 'stacks', 'crops', 'slices', 'rows', 'cols', 'compartments']
```

Comment:
This list of strings is used twice, maybe we can move it to the top as a global/constant:
`DIMENSION_LABELS = ['fovs', 'stacks', 'crops', 'slices', 'rows', 'cols', 'compartments']`